app/models/bulk_reassignment.rb

```diff
@@ -48,22 +49,24 @@ def insert_previous_assignments
         #{connection.quote(now)},
         a.bulk_assignment_id,
         a.individually_overridden,
-        visitor_supersession_id,
-        context
+        a.visitor_supersession_id,
+        a.context,
+        a.force
       FROM assignments a
       WHERE a.id #{assignment_id_clause}
     SQL
   end
 
-  def update_assignments
+  def update_assignments # rubocop:disable Metrics/AbcSize
     connection.execute(<<-SQL)
       UPDATE assignments SET
         variant = #{connection.quote(bulk_assignment.variant)},
         updated_at = #{connection.quote(now)},
         mixpanel_result = NULL,
         bulk_assignment_id = #{connection.quote(bulk_assignment)},
```

Comment:
not from this diff, but was this supposed to be `bulk_assignment.id`?

Reply:
it's the behavior of quoting an activerecord and IIRC I chose it explicitly, but it does seem a little fancy.

Reply:
yea, i suspected that, but it just seemed unnecessary to me since you can just ask for the id and it's less surprising to me that way. but i don't care that much.